### PR TITLE
Update readme with discover monitor pod

### PR DESCRIPTION
**Pull request by Kelvin-M**

### CODE REVIEW

File: charts/hlf-k8s/README.md [hunk 1]
```diff
@@ -240,6 +241,21 @@ monitor:
...
+It is a convenience feature which facilitates troubleshooting.
+
+To enable it, on the peer:
+
+```
+discover-monitor:
+   enabled: true
+```
+
+Check the pod's logs to get the list of organizations currently connected in the application channel.
```
Comment (AurelienGasser) [inline charts/hlf-k8s/README.md:257]: This is a duplicate of the first sentence

File: charts/hlf-k8s/README.md [hunk 2]
```diff
@@ -240,6 +241,21 @@ monitor:
 
 Check the pod's logs to get the list of organizations currently present in each channel.
 
+### Discover monitor pod
+
+The discover monitor pod periodically displays the list of organizations that are connected to the application channel with the hlf [service discovery](https://hyperledger-fabric.readthedocs.io/en/release-1.4/discovery-cli.html).
```
Comment (AurelienGasser) [inline charts/hlf-k8s/README.md:246]: Could you explain the difference with the monitor pod (joined v.s. connected), and in which case this is useful?

File: charts/hlf-k8s/README.md (hunk 1 above)
Comment (Kelvin-M) [reply charts/hlf-k8s/README.md:257]: I don't think so, I use the same pattern as previous :/

File: charts/hlf-k8s/README.md (hunk 2 above)
Comment (Kelvin-M) [reply charts/hlf-k8s/README.md:246]: Monitor does not show joined peer but added. I fix it.

**Comment by AurelienGasser** [comment]
I still don't find it clear what the difference between the two pods is.

Consolidating the two sections (Monitor pod / Discover monitor pod) would help with the repetitions.

It would also help with better highlighting the differences between the two

Something like

```
# Monitor pods

Two monitor pods can be used for troubleshooting:
- monitor pod: lists the orgs that **have joined** a channel
- monitor pod: lists the orgs that **have joined and are still currently connected** to a channel

```

**Comment by Kelvin-M** [comment]
@AurelienGasser Monitor pod only show orgs that have been added (with their config and secrets) in the channel. It does not mean that they join. They can join, but looking at channel configuration won't tell you that.
```md
# Monitor pods

Two monitor pods can be used for troubleshooting:
- monitor pod: lists the orgs that **are added** a channel
- discover-monitor pod: lists the orgs that **have joined and are still currently connected** to a channel
```

**Pull request merged**